src/config/backends/splash.ts

```diff
@@ -0,0 +1,187 @@
+import * as Bluebird from 'bluebird';
+import * as _ from 'lodash';
+import { fs } from 'mz';
```

Comment:
See if you can use the native fs.Promises like in https://github.com/balena-io/balena-supervisor/blob/0c7cb46d1551fb92ae7dbfe9f303181e72d739b7/src/config/backends/odmdata.ts#L2

Reply:
What is the advantage @20k-ultra ?

Reply:
The mz module was made to bring new ECMAScript features to nodejs faster then the node project implements them. One example being Promise based fs module. Since the standard lib supports Promises in the fs module theirs no need to use mz for fs. I just checked out the repo too and it was last updated in 2017 so maybe all the features it meant to modernize have been implemented.

I'm also in favour of always trying to remove dependencies :p

Reply:
I'm for removing dependencies, but I didn't end up making this change since it will mess with the tests as well since other backends use 'mz' as dependency as well. I will create an issue to study the removal of the package